setup.cfg

```diff
@@ -113,3 +113,4 @@ python-tag = py38
 omit =
     src/sentry/migrations/*
 source = .
```

Comment:
With all due respect... this all started because a completely unrelated PR wasn't passing our codecov Ci check because `if TYPECHECKING` lines were being counted, so I tweaked the settings just enough to fix that one problem, and this has now turned into a multi-day, 4-PRs-and-counting affair.

I will look into the `covdefaults` settings if I have time later this week, but also wouldn't mind some suggestions, since you're much more familiar with this stuff than I am.

Reply:
Apologies if the above comes across harshly - I don't mean it that way. This just feels like it's blown up into a much larger endeavor than I was counting on. I get what you mean about the threshold and I can fix that, and get rid of the default source. Anything else you think I should look at?

Reply:
Update: I tried `[coverage:report] fail_under = 0` and it doesn't seem to be picking it up. Any idea why?